src/Chapter1.hs

```diff
@@ -619,8 +627,11 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n =
+    let last = mod n 10
+        before_last = div (mod n 100 - last) 10
```

Comment:
Great!
For `div` the last digit won't matter, so it is okay not to subtract it in here 🙂 

Also note, that negative case is not going to work properly with `mod`, unfortunately.

And one stylistic note, In Haskell, camel case is the standard for variables 🙂
```haskell
        beforeLast = div (mod n 100) 10
```